Resources/Private/Templates/Standard/Ajax/TopicListMenu.html

```diff
@@ -52,7 +52,7 @@
                 </li>
                 <li></li>
                 <li>
-                    <f:link.action controller="Moderation" action="topicConformDelete" arguments="{topic: topic}">
+                    <f:link.action controller="Moderation" action="topicConformDelete" arguments="{topic: topic}" onclick="return confirm('{f:translate(key:'Confirm_Delete_Topic')}')">
```

Comment:
If I understand correctly, this is exactly what the `topicConformDelete` [sic!, should be `topicConf{o => i}rmDelete`] action should already be doing.